replication-api-impl/src/main/java/com/connexta/replication/api/impl/persistence/pojo/FilterIndexPojo.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.api.impl.persistence.pojo;
+
+import com.connexta.replication.api.data.FilterIndex;
+import java.time.Instant;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.springframework.data.solr.core.mapping.Indexed;
+import org.springframework.data.solr.core.mapping.SolrDocument;
+
+/**
+ * This class provides a pojo implementation for a {@link
+ * com.connexta.replication.api.data.FilterIndex} capable of reloading all supported fields for all
+ * supported versions from the database. It also provides the capability of persisting back the
+ * fields based on the latest version format.
+ */
+@SolrDocument(collection = FilterIndexPojo.COLLECTION)
+public class FilterIndexPojo extends Pojo<FilterIndexPojo> {
+
+  /**
+   * List of possible versions:
+   *
+   * <ul>
+   *   <li>1 - initial version.
+   * </ul>
+   */
+  public static final int CURRENT_VERSION = 1;
+
+  public static final String COLLECTION = "filter_index";
+
+  @Nullable
+  @Indexed(name = "modified_since", searchable = false)
+  private Instant modifiedSince;
+
+  @Nullable
```

Comment:
✏️ nit - `filterId` should come before `modifiedSince` since it is part of what uniquely identifies this index